usr/share/rear/restore/TSM/default/400_restore_with_tsm.sh

```diff
@@ -20,9 +20,10 @@ local backup_restore_log_suffix="restore.log"
 # echo -n $CONFIG_APPEND_FILES (without double quotes) is used to avoid leading and trailing spaces and newlines:
 test "$CONFIG_APPEND_FILES" && backup_restore_log_prefix=$backup_restore_log_prefix.$( echo -n $CONFIG_APPEND_FILES | tr -d -c '[:alnum:]/[:space:]' | tr -s '/[:space:]' ':_' )
 local backup_restore_log_filespace=""
+local dsmc_parallel="false"
```

Comment:
I think such an option should be user controllable
via a new `TSM_~ config variable in default.conf
https://github.com/rear/rear/blob/master/usr/share/rear/conf/default.conf#L1586
e.g. something like
```
# Restore TSM filespaces in parallel by running
# for each TSM filespace a separated 'dsmc restore' process:
TSM_DSMC_RESTORE_PARALLEL="false"
```
and then use TSM_DSMC_RESTORE_PARALLEL
instead of dsmc_parallel in the code here.